### ISSUE TITLE
ppgen assertion failed before update (code 1407)

### ISSUE THREAD

**Issue by siorai** [keybase/client#14894]
Trying desperately to add another device as this failing ( but previously verified! ) is the last device I have left... trying to note wipe keys for the 2nd in 2 years if I can help it.  

Thanks!
my log id: c9caff79570e35b57dc45e1c

**Comment by maxtaco** [comment]
Weird. I will take a look when I get s chance.

**Comment by maxtaco** [comment]
Can you upgrade your app? It is very old

**Comment by siorai** [comment]
That did it! Thanks @maxtaco 

For background: 

I have to build the client from source due to having it on my raspberry pi, I initially figured that it was probably too old a version to interact properly with current versions, but, I thought I had updated it and built a current one.... 

Obviously that wasn't the case.  As I was able to to verify my ancient version via `keybase --version`.  

That said, would it be possible in the future to maybe have the keybase client notify the user when their versions are a bit too old to properly authorize new devices?  

I imagine it probably doesn't happen that often, but between having to do a full wipe on my computer and my phone completely crapping out I had to resolve this via a small raspberry pi 3 image I had tucked away in Google Drive.  Might be useful for other users who find themselves in a similar situation. 

Thanks again.
